docs/tutorials/javascript.rst

```diff
@@ -0,0 +1,130 @@
+.. _tutorial-javascript:
+
+Kinto for JavaScript applications
+#################################
+
+Kinto is particularly relevant for the Web, thus it is very probable that you'll
+want to interact with the server using JavaScript.
+
+This tutorial presents the basics, but you can look at
+:ref:`the applications examples <app-examples>`, most of them use JavaScript.
+
+
+Offline and synchronization
+---------------------------
+
+We built :github:`kinto.js <Kinto/kinto.js>`, a JavaScript client dedicated
+to synchronizing remote records with a local database.
+
+The best way to start is :rtd:`to follow the dedicated tutorial <kintojs>` !
+
+Using an offline-first client is not mandatory, see the following section for
+a tutorial using «*vanilla JS*».
```

Comment:
I think the differentiator here isn't that it's vanilla JS, it's more that the data isn't stored locally: this is raw requests to the server.